pkgs/applications/audio/easyabc/default.nix

```diff
@@ -1,7 +1,23 @@
-{ lib, fetchFromGitHub, substituteAll, makeWrapper, python39, fluidsynth, soundfont-fluid, wrapGAppsHook, abcmidi, abcm2ps, ghostscript }:
+{ lib, fetchFromGitHub, fetchPypi, substituteAll, makeWrapper, python39, fluidsynth, soundfont-fluid, wrapGAppsHook, abcmidi, abcm2ps, ghostscript }:
 
-# requires python39 due to https://stackoverflow.com/a/71902541 https://github.com/jwdj/EasyABC/issues/52
-python39.pkgs.buildPythonApplication {
+let
+  # requires python39 due to https://stackoverflow.com/a/71902541 https://github.com/jwdj/EasyABC/issues/52
+  python = python39.override {
+    packageOverrides = self: super: {
+      self = python;
```

Comment:
I think this should be moved up one line.

Reply:
applied in https://github.com/NixOS/nixpkgs/pull/243787/commits/6857e5dff352e1e5fda1b6e2fc33b69ac3b38bd9